edgefirewall.yaml

```diff
@@ -332,6 +510,188 @@ components:
           type: boolean
         waf_enabled:
           type: boolean
+
+    Behaviors:
+      properties:
+        name:
+          type: string
+          enum: [deny, drop, set_rate_limit, set_waf_ruleset, run_function, tag_event]
+        target:
+          type: string
```

Comment:
Ok, let me clear up a doubt in the docs there is a tab that says that the behavior has an arguments this is true?

"behaviors": [
         {
           "name": "set_waf_ruleset_and_waf_mode",
           "argument": {
             "waf_mode": "blocking",
             "set_waf_ruleset_and_waf_mode": "<waf_id>"
           }
         }

Reply:
As is in docs:

"If you selected **Set Rate Limit** and **Set WAF Rule Set** you need..." - provide extra parameters in behavior, like you said - As the doc table describe: https://api.azion.com/#ff2b221b-0054-407f-aad3-39cda3a9ed0f

A supplied just a small sample, using `deny` - so it is correct.